Containerfile

```diff
@@ -4,17 +4,11 @@ MAINTAINER LSIT Systems <lsitops@lsit.ucsb.edu>
 
 USER root
 
-RUN sudo apt update ; apt install -y cargo rustc; apt-get clean
+RUN sudo apt update ; apt-get clean
```

Comment:
This line doesn't actually do anything in the end.  The apt upgrade is happening in the upstream base and this doesn't even do that now.  It just downloads the metadata and then deletes it.